exchange-v2/test/v2/ExchangeV2.rarible.test.js

```diff
@@ -62,33 +62,33 @@ contract("ExchangeV2, sellerFee + buyerFee =  6%,", accounts => {
 	beforeEach(resetState);
 
 	describe("on-chain orders", () => {
-        it("should create, update then cancel order", async () => {
```

Comment:
it would be better to fix formatting in separate PRs

Reply:
Idea: fix format in test after all merge in RPC-143-Audit-OnChainOrders